virtcontainers/store/backend.go

```diff
@@ -48,4 +48,5 @@ type backend interface {
 	delete()
 	load(item Item, data interface{}) error
 	store(item Item, data interface{}) error
+	raw(id string) (string, error)
```

Comment:
Could you please highlight in the commit message and through some comments in the code, some concrete use cases for this new API.

Reply:
I did. Please let me know if this looks better.

Reply:
Yes it helps a lot to understand :)
IMO, you should even mention
```
// For example the Firecracker code uses this API to create temp
// files under the sandbox state root path, and uses them as block
// driver backend placeholder.
```
from the commit message.